src/room/room.service.ts

```
@@ -20,15 +23,39 @@ export class RoomService {
+    } catch (e) {
+      this.logger.error(`Cannot edit the room: ${e}`);
+      throw new NotFoundException('Room not found');
+    }
+  }
+
+  async deleteRoom(id: string): Promise<{ success: boolean }> {
+    try {
+      await this.prisma.room.delete({ where: { id } });
+
+      return { success: true };
+    } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)